Formula/hivemind.rb

```diff
@@ -0,0 +1,29 @@
+class Hivemind < Formula
+  desc "The mind to rule processes of your development environment"
+  homepage "https://github.com/DarthSim/hivemind"
+  url "https://github.com/DarthSim/hivemind/archive/v1.0.tar.gz"
+  sha256 "7dde50f8f68214929f53c380ffc6311b39aad071a67bb6d94ebc3e641f78083a"
+
+  head "https://github.com/DarthSim/hivemind.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/DarthSim/hivemind/").install Dir["*"]
+    system "go", "build", "-o", "#{bin}/hivemind", "-v", "github.com/DarthSim/hivemind/"
+  end
+
+  test do
+    File.open(testpath/"Procfile", "w") do |file|
+      file.write("test: echo 'test message'")
+    end
+
+    actual = pipe_output("#{bin}/hivemind")
+    expected = "\e[1;32mtest\e[0m | \e[1mRunning...\e[0m\n" \
+               "\e[1;32mtest\e[0m | test message\n" \
+               "\e[1;32mtest\e[0m | \e[1mProcess exited\e[0m\n"
+
+    assert_equal expected, actual
```

Comment:
Just go for `assert_match "test_message", shell_output("#{bin}/hivemind"); we don't need to match all the e.g. colour escape codes.